src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
The use of the `*` selector is discouraged due to performance reasons. In this case, it is not present, but keep this in mind for future reference.